services/device-registry-jdbc/src/test/java/org/eclipse/hono/deviceregistry/jdbc/impl/AbstractJdbcBaseRegistryTest.java

```diff
@@ -69,24 +69,24 @@
 import io.vertx.junit5.VertxExtension;
 
 @ExtendWith(VertxExtension.class)
-abstract class AbstractJdbcRegistryTest {
+abstract class AbstractJdbcBaseRegistryTest {
```

Comment:
I do not quite understand why this class (and all the others) need to be renamed ...

Reply:
I change the name back to AbstractJdbcRegistryTest. 
We add the "Base" because the plan was to define the DATABASE_TYPE to abstractJDBC classes and create one per type (AbstractJdbcBaseRegistryTest, AbstractJdbcPostgresRegistryTest etc....) so that we can run unit tests for different kinds of DBs. 
We solve the DATABASE_TYPE problem by creating the JdbcPostgresDeviceManagementSearchDevicesTest  class and nun there is no need to change the name of the abstract.